admin-dev/themes/new-theme/template/components/layout/warning_messages.tpl

```diff
@@ -27,7 +27,7 @@
     <div class="alert alert-warning">
       <button type="button" class="close" data-dismiss="alert">&times;</button>
       {if count($warnings) > 1}
-        <h4>{l s='There are %d warnings:' sprintf=[count($warnings)]}</h4>
+        <h4>{l s='There are %d warnings:' sprintf=[count($warnings)] d='Admin.Notifications.Error'}</h4>
```

Comment:
Same question than for the previous occurrence.

Reply:
Sorry, I was not clear enough. I was talking about the previous error notification [above](https://github.com/PrestaShop/PrestaShop/pull/15111#discussion_r315227455).